qfieldsync/ui/cloud_projects_dialog.ui

```diff
@@ -155,14 +155,6 @@
            <string>Owner</string>
           </property>
          </column>
-         <column>
-          <property name="text">
-           <string>Local</string>
-          </property>
-          <property name="toolTip">
-           <string>Whether the file has a configured local directory</string>
-          </property>
-         </column>
```

Comment:
I would keep the tooltip that explains what the icon means somehow.

Reply:
You mean adding a tooltip for the name column? (I've kept a tooltip when you mouse hover the row)

Reply:
Shouldn't it possible to have a tooltip on the icon itself?

Reply:
AFAIK, no as it's merely a QIcon added as a Qt.DecorationRole in the item. But again, we now have the full name column for each row with the tooltip, hit area is much larger.